### DISCUSSION TITLE
labelme can not input chinese, only English

### DISCUSSION THREAD

**Discussion by cqray1990** [wkentaro/labelme#1180] [Q&A / Help]
### Provide environment information

/anaconda3/bin/python
Python 3.6.7
labelme                       5.0.1

### What OS are you using?

ubantu 16

### Describe the Bug

labelme can not input chinese, only English

**Comment by maplelost** [comment]
i use the Labelme in windows10 with Python 3.9.7 , Labelme perform correctly, also my PyQt5 == 5.15.4, if you wanna fix it you need to provide more info.
